solr/solr-ref-guide/src/jwt-authentication-plugin.adoc

```diff
@@ -161,6 +161,10 @@ Let's comment on this config:
 <12> Configure the audience claim. A token's 'aud' claim must match 'aud' for one of the configured issuers.
 <13> This issuer is auto configured through discovery, so 'iss' and JWK settings are not required
 
+=== Using non SSL URLs
+In Production you should always use SSL protected HTTPS connections since Solr is making an outbound connection,
```

Comment:
Production -> "production environments"
Remove "since Solr is making an outbound connection"?